keras/layers/normalization.py

```diff
@@ -35,6 +35,7 @@ class BatchNormalization(Layer):
         weights: Initialization weights.
             List of 2 Numpy arrays, with shapes:
             `[(input_shape,), (input_shape,)]`
+			Note that the order of this list is [Gamma, Beta, mean, std]
```

Comment:
Really sorry for the wrong pushs. I didn't see any wrong indentation in my notepad++ editor, anyway I'll check and push again.

BTW, I've translated Keras documents into Chinese. I think this Chines version documents would be helpful for those who use Keras. The homepage of this document is http://keras-cn.readthedocs.io/en/latest/ , you may want to add this link to keras.io. Thank you~ :）